wiki/Bazel-User-Guide.md

```diff
@@ -0,0 +1,277 @@
+## Table of Contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [Building the app](#building-the-app)
+- [Running Tests](#running-tests)
+  - [How to Obtain a Test Target](#how-to-obtain-a-test-target)
+  - [Common Flags](#common-flags)
+  - [Running multiple test targets](#running-multiple-test-targets)
+- [Concepts and Terminology](#concepts-and-terminology)
+- [Syncing the Project](#Syncing-the-project)
```

Comment:
This doesn't seem linkified correctly--please fix.

Reply:
Fixed.